### ISSUE TITLE
Color Picker in App Settings page is distorted

### ISSUE THREAD

**Issue by rufferson** [abranson/rockpool#17]
Seems like QtWebKit bug, would need replacement of the WebKit vs Geko - see https://github.com/tmeshkova/qmlmozbrowser as an example

**Comment by rufferson** [comment]
Localized color picker issue to jQuery-based.
Recommended by pebble Slate interface works like a charm with no issues.
jQuery generated interface (eg. hosted at http://pebbleconfig.club) seems to be wrongly evaluating viewport size while running in the WebKit - it puts statically generated inline style which sets component width to smth like 10px.
Attempted to place overriding JS snippet to resize component, however async nature of jQ requires hooking it up to the jQ's events, which is a bit problematic.
QtMozEmbed so far poses too big overhead for such a small function.
In other words - atm "wont fix" as vendor-recommended approach works fine.

**Comment by rufferson** [comment]
It seems there's more to that - eg. TH3 uses dumb-simple generated color pickers howere they are put into scrollable divs (overflow:auto) which is also not supported by qtwebview.
If one could force everyone using slate - the problem is solved. But in real world that somehow should be dealt with. So changing _wontfix_ to _will-fix-in-spare-time_

**Comment by abranson** [comment]
Now we're on QtMozEmbed, this should be fixed. Can anyone confirm? My monochrome watch doesn't give me a colour picker.

**Comment by rufferson** [comment]
Ah, yeah, all those glitches I've spotted in WebKit are working properly in Gecko, so I'm closing the case.
